### PR TITLE
Fix CMS password login issues

### LINKED ISSUE #21259: CMS connection with SQL Authentication fails after save
- Azure Data Studio Version: Insiders (v1.41.0-insider)
- OS Version: Windows 11
- Extension: CMS

Steps to Reproduce:

1. Create a new CMS connection with SQL Authentication
2. Once added, open tree again or Right click > Refresh.
3. ADS fails with below error:

image

### PR DESCRIPTION
**Pull request by cheenamalhotra**
This PR fixes #21259 

Before:

ADS_CMS_Issue

Behavior with fix:

cms-passwords

### CODE REVIEW

File: extensions/cms/src/cmsUtils.ts [hunk 1]
```diff
@@ -143,8 +143,6 @@ export class CmsUtils {
 		let toSaveCmsServers: ICmsResourceNodeInfo[] = this._registeredCmsServers.map(server => Object.assign({}, server));
 		toSaveCmsServers.forEach(server => {
 			server.ownerUri = undefined;
-			// don't save password in config
```
Comment (alanrenmsft) [inline extensions/cms/src/cmsUtils.ts:146]: if you remove this, the password will be clear text in the configuration file.

Comment (Charles-Gagnon) [reply extensions/cms/src/cmsUtils.ts:146]: Yeah, we can't do this. You'll need to persist the passwords in secret storage separately like I do in the BDC extension and then load it when you try to connect.

Saving : https://github.com/microsoft/azuredatastudio/blob/main/extensions/big-data-cluster/src/bigDataCluster/tree/controllerTreeDataProvider.ts#L154

Loading : https://github.com/microsoft/azuredatastudio/blob/main/extensions/big-data-cluster/src/bigDataCluster/tree/controllerTreeDataProvider.ts#L117

**Comment by kburtram** [comment]
@cheenamalhotra are you still planning to do something with this PR?  Or should we close until you're ready to revisit?

**Comment by cheenamalhotra** [comment]
Let me take a look at it again today now that I know a bit more in this space.. I'll close it later today if it needs significant time.

File: extensions/cms/package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "cms",
   "displayName": "%cms.displayName%",
   "description": "%cms.description%",
-  "version": "0.9.3",
```
Comment (alanrenmsft) [inline extensions/cms/package.json:5]: why skipping 0.9.4?

**Review by alanrenmsft — APPROVED** [review]

**Pull request merged**